### DISCUSSION TITLE
Edge/ Chrome Extension Policies, config via reg

### DISCUSSION THREAD

**Discussion by Lif3styl3s** [Authenticator-Extension/Authenticator#1016] [Q&A]
### Describe the issue

Hello,

we are deploying this in an enterprise and using the policies outlined [here](https://authenticator.cc/docs/en/policy-config) 

Edge has been configured here: Policies\Microsoft\Edge\3rdparty\extensions\ocglkepbibnalbgmbachknglpdipeoio\policy
Chrome has been configured here: \Policies\Google\Chrome\3rdparty\extensions\bhghoamapcdpbohphigoooaddinpkbai\policy

Both have all the policies below enabled:

**Working:**
passwordPolicy
passwordPolicyHint
feedbackURL https://*
enforceAutolock,5
enforcePassword 1
disableExport 1
disableInstallHelp 1

**Not Working**

disableBackup 1 or 0

The documentation states that this will remove access to 3rd Party Backup locations, change the value between 0, 1 does not block these actions. I would like to be able to remove 3rd Party Backup.

Installed Version is 6.3.3 (latest on the store)

Thanks

### Browser

Edge

### Browser Version

110.0.1587.50

### Extension Version

6.3.3

**Comment by Lif3styl3s** [comment]
Boolean 1/0 not true/false

**Comment by Lif3styl3s** [comment]
I have updated my issue, it  now relates to just one setting

disableBackup

**Comment by Lif3styl3s** [comment]
any update or any way of getting this working that other users have found thanks

**Comment by marcelnagyMP** [comment]
Hi, I'm trying to disable the help page on first launch but no luck.

I add to the registry:
HKEY_LOCAL_MACHINE\SOFTWARE\Policies\Google\Chrome\3rdparty\extensions\bhghoamapcdpbohphigoooaddinpkbai\disableInstallHelp
type: REG_SZ
value data: true

I also changed the type to:
type: REG_DWORD
value data: 1

Authenticator app version is 8.0.1

I couldn't find the setting in chrome://policy.

Am I doing something wrong?
Can you help me?

Thank you
